src/Featurizers/CountVectorizerFeaturizer.h

```diff
@@ -0,0 +1,481 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#pragma once
+
+#include "Components/PipelineExecutionEstimatorImpl.h"
+#include "Components/DocumentStatisticsEstimator.h"
+#include "../Traits.h"
+#include "../Strings.h"
+#include "Structs.h"
+
+namespace Microsoft {
+namespace Featurizer {
+namespace Featurizers {
+
+/////////////////////////////////////////////////////////////////////////
+///  \class         CountVectorizerTransformer
+///  \brief         Returns SparseVectorEncoding<std::uint32_t> for each unique input
+///
+class CountVectorizerTransformer : public StandardTransformer<std::string, SparseVectorEncoding<std::uint32_t>> {
+public:
+    // ----------------------------------------------------------------------
+    // |
+    // |  Public Types
+    // |
+    // ----------------------------------------------------------------------
+    using BaseType                           = StandardTransformer<std::string, SparseVectorEncoding<std::uint32_t>>;
+    using IndexMapType                       = std::unordered_map<std::string, std::uint32_t>;
+    using AnalyzerMethod                     = Components::Details::DocumentStatisticsTrainingOnlyPolicy::AnalyzerMethod;

```

Comment:
>Components::Details::DocumentStatisticsTrainingOnlyPolicy::AnalyzerMethod; [](start = 47, length = 74)

We should not reference members in a details namespace - this enum should be moved up a level.